filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsPathBuilderFactory.scala

```diff
@@ -22,10 +23,11 @@ class DrsPathBuilderFactory(globalConfig: Config, instanceConfig: Config, single
 
   private lazy val googleConfiguration: GoogleConfiguration = GoogleConfiguration(globalConfig)
   private lazy val scheme = instanceConfig.getString("auth")
-  private lazy val googleAuthMode = googleConfiguration.auth(scheme) match {
-    case Valid(auth) => auth
-    case Invalid(error) => throw new RuntimeException(s"Error while instantiating DRS path builder factory. Errors: ${error.toString}")
-  }
+
+  // For Azure support
+  private val workflowExecutionIdentityKey = "workflow_execution_identity"
```

Comment:
My read of BT-442 is that `workflow_execution_identity` actually doesn't point to the right value to use here; we should actually use the value being added in BT-442. If so, I see this as okay for this PR (enables this code to function if the UAMI client ID is supplied though `workflow_execution_identity`) but should be updated with BT-442. Am I understanding this correctly?

Reply:
Yup, that is correct! Whichever of us merges second can fix it.